hnn_core/tests/test_network.py

```diff
@@ -517,8 +517,14 @@ def test_network():
 
     # Test removing connections from net.connectivity
     # Needs to be updated if number of drives change in preceeding tests
-    net.clear_connectivity()
-    assert len(net.connectivity) == 50
+
+    # Testing deleting a custom number of drives
+    all_drives = net.get_external_drive_names()
+    drives_to_be_deleted = all_drives[0:2]  # Deleting 2 drives in this test
+    net.clear_drives(drive_names=drives_to_be_deleted)
+    assert len(net.connectivity) == 15
```

Comment:
I just printed net.connectivity values and got it from there. But I will check out what net.connectivity is and figure out another way to find the values

Reply:
`pick_connection` may be helpful. You basically want to check how many connections the drive had before and subtract that from the total number of connections.

Reply:
I wrote my own function for getting connections of an external drive but will move to pick connection.

Also I found a bug in clear_connectivity function. Basically it will work correctly when no arguments are passed, but when drive names are passed to it, it deletes all connections other than the ones which were intended. Basically reverse behavior.

Reply:
It will be more clear once I push the commit.